src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose.cypress.spec.js

```diff
@@ -4,7 +4,10 @@ import PatientComposePage from './pages/PatientComposePage';
 import requestBody from './fixtures/message-compose-request-body.json';
 import { AXE_CONTEXT, Locators } from './utils/constants';
 
-describe('Secure Messaging Compose', () => {
+// Skip in CI due to flakiness
+const testSuite = Cypress.env('CI') ? describe.skip : describe;
+
+testSuite('Secure Messaging Compose', () => {
```

Comment:
@fazilqa for your awareness. We may have to review what is causing it to be flaky